db/schema.rb

```diff
@@ -1117,13 +1131,13 @@
     t.index ["arm_id"], name: "index_visit_groups_on_arm_id"
   end
 
-  create_table "visits", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "visits", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8_unicode_ci", force: :cascade do |t|
     t.integer "quantity", default: 0
     t.string "billing"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "deleted_at"
-    t.integer "research_billing_qty", default: 0
+    t.bigint "research_billing_qty", default: 0
```

Comment:
Not sure why this one changed

Reply:
(Same as above, no changes made to or associated with 'research_billing_qty', db/schema.rb copied from git repo)